README.md

```diff
@@ -36,7 +36,7 @@ Base requirements:
 
 ## Linux
 
-On a **Debian** based operation execute the following steps, after cloning the repository:
+On a **Debian** based operation system execute the following steps, after cloning the repository:
```

Comment:
And of course the file's name is not README.txt but README.md, sorry for that :)